src/plugins/workspace/public/plugin.ts

```diff
@@ -111,17 +125,25 @@ export class WorkspacePlugin
       this.registeredUseCases$,
     ]).subscribe(([currentWorkspace, registeredUseCases]) => {
       if (currentWorkspace) {
-        const isAllUseCase =
-          getFirstUseCaseOfFeatureConfigs(currentWorkspace.features || []) === ALL_USE_CASE_ID;
+        const workspaceUseCase = getFirstUseCaseOfFeatureConfigs(currentWorkspace.features || []);
+        const isAllUseCase = workspaceUseCase === ALL_USE_CASE_ID;
         this.appUpdater$.next((app) => {
-          // When in all workspace, the home should be replaced by workspace detail page
+          // When in all workspace, the home should be replaced by workspace overview page
           if (app.id === 'home' && isAllUseCase) {
             return { navLinkStatus: AppNavLinkStatus.hidden };
           }
 
-          // show the overview page in all use case
-          if (app.id === WORKSPACE_DETAIL_APP_ID && isAllUseCase) {
-            return { navLinkStatus: AppNavLinkStatus.visible };
+          // disable essential overview page for workspace use case is not essential
+          if (
+            app.id === ESSENTIAL_OVERVIEW_APP_ID &&
+            workspaceUseCase !== WORKSPACE_USE_CASES.essentials.id
+          ) {
+            return { status: AppStatus.inaccessible };
+          }
+
+          // disable analytics(All) overview page for workspace use case is not analytics(All)
+          if (app.id === ANALYTICS_OVERVIEW_ALL_APP_ID && workspaceUseCase !== ALL_USE_CASE_ID) {
+            return { status: AppStatus.inaccessible };
```

Comment:
I think we just need to disable overview pages when in all use case.

Reply:
essentials user case is disabled in all use case, for others they should still available.

Reply:
![image](https://github.com/user-attachments/assets/c6e74bb6-260c-42a7-bc55-cd867c6d6d19)

click on those card will navigates to each use case overview page

Reply:
I see, thanks for the clarification.